packages/wallets/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -1,16 +1,20 @@
-import React, { useCallback, useEffect, useState } from 'react';
+import React, { lazy, Suspense, useCallback, useEffect, useState } from 'react';
 import { useAvailableMT5Accounts } from '@deriv/api-v2';
 import { ModalStepWrapper, WalletButton } from '../../../../components/Base';
+import { Loader } from '../../../../components/Loader';
 import { useModal } from '../../../../components/ModalProvider';
 import useDevice from '../../../../hooks/useDevice';
 import { DynamicLeverageContext } from '../../components/DynamicLeverageContext';
 import { MarketTypeDetails, PlatformDetails } from '../../constants';
-import { Verification } from '../../flows/Verification';
 import { DynamicLeverageScreen, DynamicLeverageTitle } from '../../screens/DynamicLeverage';
 import { JurisdictionScreen } from '../../screens/Jurisdiction';
 import { MT5PasswordModal } from '..';
 import './JurisdictionModal.scss';
 
+const LazyVerification = lazy(
+    () => import(/* webpackChunkName: "wallets-verification-flow" */ '../../flows/Verification/Verification')
+);
```

Comment:
Verification flow:
https://github.com/binary-com/deriv-app/assets/103016120/69b9df39-5489-44dd-8416-53cfbfbc8539

Reply:
Lazy loading verification flow will shave off 3MB of unnecessary chunks for every page